libethdrivers/src/plat/zynq7000/zynq7000.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright 2014, NICTA
```

Comment:
Not sure why this copyright is dropped. There are large amounts of new implementation, but also large amounts of unchanged code and code with minor name/formatting changes

Reply:
You are right. We did a lot of commit squashing with these files. Feel free to reinsert it!